packages/next/src/exports/views.ts

```diff
@@ -1,4 +1,4 @@
 export { EditView } from '../views/Edit/index.js'
-export { DefaultListView } from '../views/List/Default/index.js'
+export { DefaultListView as ListView } from '../views/List/Default/index.js'
```

Comment:
This looks correct but do the same for the `EditView` above ^, i.e. drill it down to the default